subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1638,6 +1698,24 @@ static void le_iso_read_test_counters(struct net_buf *buf, struct net_buf **evt)
 }
 #endif /* CONFIG_BT_CTLR_CENTRAL_ISO || CONFIG_BT_CTLR_PERIPHERAL_ISO */
 
+#if defined(CONFIG_BT_CTLR_CENTRAL_ISO) || \
+	defined(CONFIG_BT_CTLR_PERIPHERAL_ISO) || \
+	defined(CONFIG_BT_CTLR_LE_FEATURE)
```

Comment:
Similar comment regarding when this should be compiled

Reply:
Done